modules/doc/content/python/testers/Exodiff.md

```diff
@@ -0,0 +1,41 @@
+# Exodiff
```

Comment:
Might be a good idea to mention in this page how to generate a custom cmp file:
```
${MOOSE_DIR}/framework/contrib/exodiff/exodiff -summary moose_simulation_out.e
```
and a listing of what one of those looks like. I don't remember seeing that anywhere in our current doco, and it would fit great here.

Reply:
Addressed. Thanks!